datasets/titanic/data/gender_maps.csv

```diff
@@ -1,3 +1,3 @@
-key,label,url
-0,male,wd:Q6581097
```

Comment:
nit: wd:Q6581097 is not an expanded form URL. Maybe call the colum wikidata_id or sth?